services/controllerhost/queueDepth.go

```diff
@@ -93,16 +95,32 @@ type queueDepthCalculator struct {
 	*extentStateMonitor
 	ll                      bark.Logger
 	destinationExtentsCache replicaStatsMRUCache // MRU cache of extent replica stats
-	dlqExtentsCache         replicaStatsMRUCache // MRU cache of dlq extent replica stats
+	dstCoverMap             extentCoverMap
+	dlqCoverMap             extentCoverMap
 }
 
 func newQueueDepthCalculator(m *extentStateMonitor) *queueDepthCalculator {
 	return &queueDepthCalculator{
 		extentStateMonitor:      m,
-		destinationExtentsCache: *newReplicaStatsMRUCache(m.context.mm, m.context.log.WithField(`module`, `ReplicaStatsMRUCache`)),
-		dlqExtentsCache:         *newReplicaStatsMRUCache(m.context.mm, m.context.log.WithField(`module`, `ReplicaStatsMRUCacheDLQ`)),
-		ll:                      m.context.log.WithField(`module`, `queueDepth`),
+		destinationExtentsCache: *newReplicaStatsMRUCache(m.context.mm, m.context.log.WithField(`module`, `ReplicaStatsMRUCache`), queueDepthCacheLimit),
```

Comment:
I bet this will make the diff incomprehensible, so I'll make a 2nd diff to do rename/restructuring.